models/fivetran_log__connector_status.sql

```diff
@@ -1,14 +1,19 @@
 with connector_log as (
 
-    select *
+    select *,
+        sum( case when event_subtype in ('sync_start') then 1 else 0 end) over ( partition by connector_id 
+            order by created_at rows unbounded preceding) as sync_batch_id
     from {{ ref('stg_fivetran_log__log') }}
 
     -- only looking at errors, warnings, and syncs here
     where event_type = 'SEVERE'
         or event_type = 'WARNING'
         or event_subtype like 'sync%'
+        or (event_subtype = 'status' 
+            and {{ fivetran_utils.json_parse(string="message_data", string_path=["status"]) }} ='SUCCESSFUL')
         or (event_subtype = 'status' 
             and {{ fivetran_utils.json_parse(string="message_data", string_path=["status"]) }} ='RESCHEDULED'
```

Comment:
wondering if we should combine these two to only call `json_parse` once 

so like 
```suggestion
        or (event_subtype = 'status' 
            and {{ fivetran_utils.json_parse(string="message_data", string_path=["status"]) }}  in ('SUCCESSFUL', 'RESCHEDULED'))
```

Reply:
Way more elegant with this method. Thanks!

Reply:
Actually I just tried this and found that it was not working as intended. The `SUCCESSFUL` status events do not also have the `intended behavior` message attached to them. Because of that, when we combine these json parse queries, they actually do not bring through the successful events. 

Because of that, we should keep it as is.